src/modules/esl-media/core/esl-media.ts

```diff
@@ -381,6 +404,10 @@ export class ESLMedia extends ESLBaseElement {
     }
   }
 
+  protected isAcceptableTarget(target: EventTarget | null): boolean {
+    return isElement(target) && target.contains(this);
+  }
```

Comment:
Move to esl-utils/dom/traversing, please.
Use something like `isContain(container, element)`
PS: `isRelativeNode` could reuse `isContain` then